mrmustard/lab_dev/circuit_components.py

```diff
@@ -317,6 +319,60 @@ def from_quadrature(
         BBBB = QtoB_ib @ (QtoB_ik @ QQQQ @ QtoB_ok) @ QtoB_ob
         return cls._from_attributes(BBBB.representation, wires, name)
 
+    def to_quadrature(self, phi: float = 0.0) -> CircuitComponent:
+        r"""
+        Returns a circuit component with the quadrature representation of this component
+        in terms of A,b,c.
+
+        Args:
+            phi: The quadrature angle. ``phi=0`` corresponds to the x quadrature,
+                    ``phi=pi/2`` to the p quadrature. The default value is ``0``.
+        Returns:
+            A circuit component with the given quadrature representation.
+        """
+        if isinstance(self.representation, Fock):
```

Comment:
We now have the option to convert a Fock object to Bargmann and then use BtoQ, but I am not sure if we want to do that?

Reply:
Maybe we could with a warning?

Reply:
Why would we need a warning? I think it's fine without.